wasm/index.go

```diff
@@ -126,28 +126,29 @@ func (m *Module) populateTables() error {
 	for _, elem := range m.Elements.Entries {
 		// the MVP dictates that index should always be zero, we should
 		// probably check this
-		if int(elem.Index) >= len(m.TableIndexSpace) {
+		if elem.Index >= uint32(len(m.TableIndexSpace)) {
```

Comment:
In 32bit system, `int` may only have 32 bits, so if `elem.Index` is too large, `int(elem.Index)` may be negative, and pass through this test, cause panic in the following operation.